### DISCUSSION TITLE
AudioGridder appears as "AGidder (inst)" under Network, and as "AudioGriddderInst" under Other

### DISCUSSION THREAD

**Discussion by Adam-05** [apohl79/audiogridder#1335] [Q&A]
Why does AG appear as "AGidder (inst)" under Network, and as "AudioGriddderInst" under Other, in Cubase 12?
What's the difference?

**Comment by psyfer955** [comment]
does one have the vst3 marker on the right hand side?
<img width="348" alt="Screenshot 2024-10-15 at 2 30 45 PM" src="https://github.com/user-attachments/assets/a7af59cb-9ca1-49bf-99d2-88ad44009f39">
